lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.mel

```diff
@@ -233,8 +233,13 @@ proc fileOptionsTabPage(string $tabLayout)
         -cc ("fo_changeReferenceNamespaceOption(\"" + $action + "\")")
         -annotation `uiRes("m_fileOptions.kNamespacesEnsureUniqueNodeNames")`
         useNamespacesGrp;
-    //All offlineMode is in namespace, so we set the useNamespacesGrp be true and gray
-    frameLayout -edit -expandCommand ("fo_NamespaceOptionExpand(\"" + $action + "\")") namespaceOptionFrame;
+
+    // For Usd Add Maya Reference we always want the "use namespaces" option to be true.
+    // So hide this checkbox.
+    // Note: we must leave all the controls for the non-useNamepsaces option as the
+    //       callbacks in Maya expect the controls to be there.
+    checkBoxGrp -edit -v1 true -manage false -visible false useNamespacesGrp;
+    frameLayout -edit -expandCommand ("fo_NamespaceOptionExpand(\"" + $action + "\"); checkBoxGrp -edit -v1 true -manage false -visible false useNamespacesGrp") namespaceOptionFrame;
```

Comment:
At first I had removed all the other controls (that are shown when use namespace is off), but that gave me all sorts of script errors from the fileOptions.mel in Maya since it refers to these controls.

So now I left all of them, but make sure that "use namespaces" is always checked (and hidden).